jbpm-system-test/jbpm-springboot-mssql-test/src/test/java/org/kie/server/springboot/samples/KieServerMSSQLTest.java

```diff
@@ -0,0 +1,172 @@
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+import org.springframework.test.context.TestPropertySource;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+import org.testcontainers.containers.MSSQLServerContainer;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+import static org.springframework.test.annotation.DirtiesContext.ClassMode.AFTER_CLASS;
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@SpringBootTest(classes = {JBPMApplication.class}, webEnvironment = WebEnvironment.RANDOM_PORT)
+@TestPropertySource(locations = "classpath:application-test-mssql.properties")
+@DirtiesContext(classMode= AFTER_CLASS)
+public class KieServerMSSQLTest {
+
+    private static final Logger logger = LoggerFactory.getLogger(KieServerMSSQLTest.class);
+
+    static final String PATH = "src/test/resources/kjars/";
+    
+    static final String ARTIFACT_ID = "evaluation";
+    static final String GROUP_ID = "org.kie.server.springboot.samples";
+    static final String VERSION = "1.0.0";
+
+    static final String ALIAS = "eval";
+    static final String CONTAINER_ID = "evaluation";
+    static final String PROCESS_ID = "evaluation";
+
+    static final String user = "john";
+    static final String password = "john@pwd1";
+
+    @LocalServerPort
+    private int port;
+
+    private KieServicesClient kieServicesClient;
+
+    static final MSSQLServerContainer<?> MSSQL_CONTAINER;
+    
+    static {
+        MSSQL_CONTAINER = new MSSQLServerContainer<>(System.getProperty("org.kie.samples.image.sqlserver","mcr.microsoft.com/mssql/server:2019-latest"))
+                                 .acceptLicense();
+        
+        MSSQL_CONTAINER.withInitScript("sql/sqlserver-springboot-jbpm-schema.sql");
+        MSSQL_CONTAINER.start();
+    }
+    
+    @BeforeClass
+    public static void generalSetup() {
+        logger.info("mssql started at "+MSSQL_CONTAINER.getJdbcUrl());
+        
+        KieJarBuildHelper.createKieJar(PATH + ARTIFACT_ID);
+    }
+
+    @AfterClass
+    public static void generalCleanup() {
+        System.clearProperty(KieServerConstants.KIE_SERVER_MODE);
+        MSSQL_CONTAINER.stop();
+    }
+
+    @DynamicPropertySource
+    public static void registerDbProperties(DynamicPropertyRegistry registry) {
+        registry.add("spring.datasource.url", MSSQL_CONTAINER::getJdbcUrl);
+        registry.add("spring.datasource.password", MSSQL_CONTAINER::getPassword);
+        registry.add("spring.datasource.username", MSSQL_CONTAINER::getUsername);
+    }
+    
+    @Before
+    public void setup() throws InterruptedException {
```

Comment:
This exception is not thrown within this method.

Reply:
done